navigation/MasterView.tsx

```
@@ -4,19 +1,271 @@
+  const handleAppStateChange = useCallback(
+    async (nextAppState: AppStateStatus | undefined) => {
+      if (wallets.length === 0) return;
+      if ((appState.current.match(/background/) && nextAppState === 'active') || nextAppState === undefined) {
+        setTimeout(() => A(A.ENUM.APP_UNSUSPENDED), 2000);
+        updateExchangeRate();
+        const processed = await processPushNotifications();
+        if (processed) return;
+        const clipboard = await BlueClipboard().getClipboardContent();
+        const isAddressFromStoredWallet = wallets.some(wallet => {
+          if (wallet.chain === Chain.ONCHAIN) {
+            return wallet.isAddressValid && wallet.isAddressValid(clipboard) && wallet.weOwnAddress(clipboard);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)